content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -91,7 +91,7 @@ To see which admission plugins are enabled:
 kube-apiserver -h | grep enable-admission-plugins
 ```
 
-In 1.18, they are:
+In 1.19, they are:
```

Comment:
Hello, I am wondering if the content below (list) has changed since 1.18? 
If not, it might be better to say since 1.18 or similar instead of changing the version here.
If this is just a mistake that was missed during the 1.19 docs change, I'm happy to approve it!

Reply:
Anyone can tech review it somehow?

Reply:
I checked on my side and there are the same on 1.18 and 1.19 🙂

Reply:
As the documentation is versioned, do we really need to quote version 1.18? 🤔  Is there any guidelines about that?

Reply:
That's a very good point, I don't think the version is even necessary here then? I didn't see any guidelines in the documentation style guide, but @sftim @kbhawkey @celestehorgan may be able to answer how to best handle this

Reply:
Hi @ebriand .
I suggest to create a general statement before listing the enabled admission plugins.

The text could be:

To see which admission plugins are enabled, run:

`kube-apiserver -h | grep enable-admission-plugins`

The output is similar to:

```
NamespaceLifecycle, LimitRanger, ServiceAccount, TaintNodesByCondition, Priority, DefaultTolerationSeconds, DefaultStorageClass, StorageObjectInUseProtection, PersistentVolumeClaimResize, RuntimeClass, CertificateApproval, CertificateSigning, CertificateSubjectRestriction, DefaultIngressClass, MutatingAdmissionWebhook, ValidatingAdmissionWebhook, ResourceQuota
```

Reply:
Hi @kbhawkey 🙂  I changed the wording to not quote the current version, WDYT?